libs/core/src/lib/utils/pipes/search-highlight.pipe.ts

```diff
@@ -4,7 +4,10 @@ import { Pipe, PipeTransform } from '@angular/core';
     name: 'highlight'
 })
 export class SearchHighlightPipe implements PipeTransform {
-    transform(value: string, args: string, active: boolean = true): string {
+    transform(value: string, args: string, active: boolean = true, filterHighlight?: boolean): string {
```

Comment:
If this pipe is bascially only for highlight filters, why do you need filterHighlight argument?